fastcrypto-zkp/src/bn254/poseidon/mod.rs

```diff
@@ -1,65 +1,89 @@
 // Copyright (c) 2022, Mysten Labs, Inc.
 // SPDX-License-Identifier: Apache-2.0
 
+use crate::bn254::poseidon::constants::{
```

Comment:
Does fmt complain? if we order them properly? alternatively we could use U01, U02 etc. In any case, minor.

Reply:
I did the quick-fix of replacing the imports with a `*`.